documentation/modules/oauth/con-oauth-authorization-keycloak-authorization-services.adoc

```diff
@@ -0,0 +1,199 @@
+// Module included in the following module:
+//
+// proc-oauth-authorization-broker-config.adoc
+
+[id='con-oauth-authorization-keycloak-authorization-services_{context}']
+= Manage policies and permissions in Keycloak Authorization Services
```

Comment:
```suggestion
= Managing policies and permissions in Keycloak Authorization Services
```

Reply:
Could we introduce this chapter with a line to say what we'll be looking at and why? "To use Keycloak with Strimzi, Keycloak Authorization Services must be mapped to Kafka's authorization model so that all possible Kafka permissions and resources are represented in Keycloak authorization scopes and policies."  Or similar.